lib/livebook/runtime/erl_dist/runtime_server.ex

```diff
@@ -644,4 +645,22 @@ defmodule Livebook.Runtime.ErlDist.RuntimeServer do
       end)
     end)
   end
+
+  defp normalize_path_env(envs, state) do
+    idx = Enum.find_index(envs, &(elem(&1, 0) == "PATH"))
+
+    if idx do
+      {{"PATH", path}, envs} = List.pop_at(envs, idx)
```

Comment:
You can do something like this:

```elixir
Enum.map(envs, fn
  {"PATH", _} ->
    # normalize

  other ->
    other
end)
```

And resetting back to `stat.initial_path` should happen on `:delete_system_envs` when deleting path :)